Gemfile

```diff
@@ -25,91 +25,93 @@ def location_for(place_or_version, fake_version = nil)
   end
 end
 
+# Gems needed by other groups
+gem 'rake', :require => false
+
 group :development do
-  gem 'rake',                         :require => false
```

Comment:
I added the rake gem to the default group because the "system_tests" group only requires rake and no other gems that are part of the "development" group.